.github/workflows/reusable_workflow_test.yml

```
@@ -25,11 +25,11 @@ jobs:
         with:
           node-version: 18
           cache: 'yarn'
-      - run: yarn install --frozen-lockfile --ignore-scripts
+      - run: YARN_ENABLE_SCRIPTS=false yarn install --immutable
 
       - name: Run tests for all workspaces
         if: ${{ !inputs.workspace }}
-        run: yarn workspaces run test:ci
+        run: yarn workspaces foreach run test:ci
```

```suggestion
        run: yarn run test:ci
```
давай так?
вижу в рутовом `package.json` уже задан `foreach` эта команда